src/components/search-history/search-history.component.tsx

```diff
@@ -34,8 +39,11 @@ const headers = [
 
 export const SearchHistory = () => {
   return (
-    <div>
-      SearchHistory
+    <div className={styles.container}>
+      <div className={styles.header}>
+        <p className={styles.heading}>Search History</p>
```

Comment:
Shouldn't you be using the translation framework instead of hard coding strings?

Reply:
I added them now